### PR TITLE
Fix database port script to work with new event_search table

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: scripts/synapse_port_db [hunk 1]
```diff
@@ -229,19 +230,51 @@ class Porter(object):
             if rows:
                 next_chunk = rows[-1][0] + 1
 
-                self._convert_rows(table, headers, rows)
+                if table == "event_search":
+                    # We have to treat event_search differnetly since it has a
```
Comment (NegativeMjark) [inline scripts/synapse_port_db]: spelling differently

**Comment by NegativeMjark** [comment]
LGTM other than the spelling. I wonder if it would be simpler to leave it to a background update task like the schema update script does.

**Comment by erikjohnston** [comment]
@NegativeMjark I thought about that, but then I concluded it would be simpler and easier to understand to just port all the data across; otherwise I would have to start fiddling with the actual data to ensure it was all consistent, etc.

**Comment by NegativeMjark** [comment]
LGTM

**Pull request merged**